domain/snyk/delta/findings_differ.go

```diff
@@ -0,0 +1,49 @@
+/*
+ * © 2024 Snyk Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package delta
+
+var _ Differ = (*FindingsDiffer)(nil)
+
+type Differ interface {
+	Diff(baseIssueList, currentIssueList []Identifiable) []Identifiable
+}
+
+type FindingsDiffer struct {
```

Comment:
I'd like a New func for instance creation, but up to you

Reply:
I had a new func before and decided to change it after. 
Reason was that I would like delta.FindingsDiffer to be reusable for other products as well. 

I have the assumption that in some cases we won't need for example a matcher or an enricher.
That's why I thought options pattern might fit. wdyt?